docs/docs/module_guides/observability/index.md

```diff
@@ -116,6 +117,39 @@ llama_index.core.set_global_handler("arize_phoenix")
 - [Auto-Retrieval Guide with Pinecone and Arize Phoenix](https://docs.llamaindex.ai/en/latest/examples/vector_stores/pinecone_auto_retriever/?h=phoenix)
 - [Arize Phoenix Tracing Tutorial](https://colab.research.google.com/github/Arize-ai/phoenix/blob/main/tutorials/tracing/llama_index_tracing_tutorial.ipynb)
 
+### Literal AI
+
+[Literal AI](https://literalai.com/) is the go-to LLM evaluation and observability solution, enabling engineering and product teams to ship LLM applications reliably, faster and at scale. This is possible through a collaborative development cycle involving prompt engineering, LLM observability, LLM evaluation and LLM monitoring. Conversation Threads and Engine Runs can be automatically logged on Literal AI.
+
+#### Usage Pattern
+
+- Install the Literal AI Python SDK with `pip install literalai`
+- On your Literal AI project, go to **Settings** and grab your API key
+- If you are using a self-hosted instance of Literal AI, also make note of its base URL
+
+Then add the following lines to your applicative code :
+
+```python
+from literalai import LiteralClient
+from llama_index.core import Document, VectorStoreIndex
+
+# By default, the Literal AI client will initialize using the following environment variables :
+# LITERAL_API_KEY
+# LITERAL_API_URL
+literalai_client = LiteralClient()
+
+# You can also explicitly provide the configuration keys like so :
+# literalai_client = LiteralClient(api_key="lsk_xxxxx", url="http://localhost:3000")
+
+literalai_client.instrument_llamaindex()
```

Comment:
Just to confirm, is this using the new `instrumentation` logic with event and span handlers? Or the older callback library?

Reply:
Hi @logan-markewich , indeed this is using the new `instrumentation` module (Event + Span handlers)